tailcfg/tailcfg.go

```diff
@@ -1369,3 +1373,82 @@ type SetDNSRequest struct {
 	// Value is the value to add.
 	Value string
 }
+
+// SSHPolicy is the policy for how to handle incoming SSH connections
+// over Tailscale.
+type SSHPolicy struct {
+	// Rules are the rules to process for an incoming SSH
+	// connection. The first matching rule takes its action and
+	// stops processing further rules.
+	Rules []*SSHRule
```

Comment:
given you're naming fields inside SSHRule, name this "rules"?

Reply:
done